test/current_deployed_version.py

```diff
@@ -1,30 +1,30 @@
 #!/usr/bin/env python
 """
 Parses the TSV file {mint_deployment_dir}/{env}.tsv and returns
-the version from the first line in the column named {pipeline_name}.
+the version from the first line in the column named {component_name}.
 For example, given the following TSV:
-10x	ss2
-1.0.0	2.0.0
-0.9.3	1.2.3
+lira	10x	ss2
+0.1.2	1.0.0	2.0.0
+0.1.1 0.9.3	1.2.3
 
-If pipeline_name is 10x, returns "1.0.0".
-If pipeline_name is ss2, returns "2.0.0".
+If component_name is 10x, returns "1.0.0".
+If component_name is ss2, returns "2.0.0".
 
```

Comment:
Maybe it is worth to also add `If component_name is lira, returns "0.1.2".` here. Also, as it shows here, https://github.com/HumanCellAtlas/dcp/blob/92cdf58f75eb31f46381d732f6a30c79f826de12/components.yml#L13, I labeled our current deployed listener with `0.2.0` since we did a lot of refactoring work based on the previous listener, and it is still [backward compatible](https://www.jvandemo.com/a-simple-guide-to-semantic-versioning/).